### PR TITLE
Fix broken link to OpenShift Pipeline documentation

### PR DESCRIPTION
**Pull request by anithapriyanatarajan**
# Changes

Updated the correct documentation to OpenShift Pipelines.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Run `make test lint` before submitting a PR
- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes
```
NONE
```

### CODE REVIEW

**Comment by anithapriyanatarajan** [comment]
Does this PR introduce a user-facing change?

NONE

**Comment by anithapriyanatarajan** [comment]
> @anithapriyanatarajan: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

/release-note-none

**Review by khrm — COMMENTED** [review]
/ok-to-test

**Review by khrm — APPROVED** [review]

**Comment by khrm** [comment]
@anithapriyanatarajan You have to sign CLA also.

File: operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml [hunk 1]
```diff
@@ -293,7 +293,7 @@ spec:
     - Tekton Results (tech-preview): v0.10.0
 
     ## Getting Started
-    In order to get familiar with _OpenShift Pipelines_ concepts and create your first pipeline, follow the [OpenShift Pipelines Docs](https://docs.openshift.com/container-platform/OPENSHIFT_DOCS_VERSION/cicd/pipelines/creating-applications-with-cicd-pipelines.html).
+    In order to get familiar with _OpenShift Pipelines_ concepts and create your first pipeline, follow the [OpenShift Pipelines Docs](https://docs.openshift.com/pipelines/PIPELINES_DOC_VERSION/about/understanding-openshift-pipelines.html).
```
[the remark this answers is not in the document]
Comment (piyush-garg) [reply operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml:296]: And the same link as before is 

```https://docs.openshift.com/pipelines/1.14/create/creating-applications-with-cicd-pipelines.html```

[the remark this answers is not in the document]
Comment (anithapriyanatarajan) [reply operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml]: Resolved.

**Review by jkandasa — APPROVED** [review]

**Comment by jkandasa** [comment]
Thanks for the contribution to the operator @anithapriyanatarajan :tada:

**Pull request merged**